vault/resource_ssh_secret_backend_role_test.go

```diff
@@ -84,19 +86,10 @@ func TestAccSSHSecretBackendRole(t *testing.T) {
 				checks = append(checks, initialCheckFuncs...)
 			}
 
-			meta := testProvider.Meta().(*provider.ProviderMeta)
-			isVaultVersion112 := meta.IsAPISupported(provider.VaultVersion112)
-			if isVaultVersion112 {
-				if isUpdate {
-					checks = append(checks,
-						resource.TestCheckResourceAttr(resourceName, "allowed_domains_template", "true"),
-					)
-				} else {
-					checks = append(checks,
-						resource.TestCheckResourceAttr(resourceName, "allowed_domains_template", "false"),
-					)
-				}
-			}
+			// can append version-dependent checks here:
+			// meta := testProvider.Meta().(*provider.ProviderMeta)
```

Comment:
Is this a placeholder comment or should this be uncommented?

Reply:
This code is a bit tricky to wrap your head around the first time you see it in my opinion. So I thought I should leave some help on how one might make changes to it. I am happy to remove it though if you think it makes it more confusing. Your comment makes me think it may be better to remove it.

Reply:
Ah gotcha, I'm not opposed to adding some guidance here

Would it make sense to maybe add something like `for example` somewhere in the comment? I initially did not recognize it as an example, so got confused 😅